src/components/UserProfile.jsx

```diff
@@ -28,12 +30,33 @@ export default function UserProfile({
   someoneElse,
   noCollaborate = false,
 }) {
-  const { data: sightingsData } = useGetUserSightings(userId);
+  const {
+    data: sightingsData,
+    loading: sightingsLoading,
+  } = useGetUserSightings(userId);
   const intl = useIntl();
   const [editingProfile, setEditingProfile] = useState(false);
   const metadataSchemas = useUserMetadataSchemas(userId);
-  const { data: agsData } = useGetUserUnprocessedAssetGroupSightings(
-    userId,
+  const {
+    data: agsData,
+    loading: agsLoading,
+  } = useGetUserUnprocessedAssetGroupSightings(userId);
+
+  const [localAgsLoading, setLocalAgsLoading] = useState(true);
+  const [localSightingsLoading, setLocalSightingsLoading] = useState(
+    true,
+  );
+  useEffect(
+    () => {
+      setLocalAgsLoading(agsLoading);
+    },
+    [agsLoading],
+  );
+  useEffect(
+    () => {
+      setLocalSightingsLoading(sightingsLoading);
```

Comment:
what I am trying to figure out is: in what situation does `localSightingsLoading !== sightingsLoading`?

Reply:
I don't think that situation exists either. Just some overkill.